src/main/java/lotto/model/LottoMachine.java

```diff
@@ -0,0 +1,27 @@
+package lotto.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import lotto.policy.LottoNumberPolicy;
+
+public class LottoMachine {
+    private static final int MIN = LottoNumberPolicy.MIN_NUMBER.number();
+    private static final int MAX = LottoNumberPolicy.MAX_NUMBER.number();
+    private static final int SCALE = LottoNumberPolicy.NUMBER_SCALE.number();
+
+    public Lotto buyLotto() {
+        return new Lotto(randomNumbers());
+    }
+
+    private List<Integer> randomNumbers() {
+        List<Integer> numbers = Randoms.pickUniqueNumbersInRange(MIN, MAX, SCALE);
+        List<Integer> sortedNumbers = new ArrayList<>(numbers);
+        Collections.sort(sortedNumbers);
+        return sortedNumbers;
```

Comment:
```suggestion
        Collections.sort(numbers);
        return new ArrayList<>(numbers);
```
코드가 많이 복잡하진 않으니 리턴값을 간결하게 작성해도 괜찮을 거 같아요